rollup.config.js

```diff
@@ -1,7 +1,7 @@
 import {builtinModules} from 'module';
 import {dependencies} from './package.json';
 
-export default {
+const configuration = {
```

Comment:
I know (and agree) that `default exports` are evil. But what is the sense to create named default exports instead of just default exports? 😮

Reply:
XO is opinionated, so we might disagree with some rules, that it introduces. I personally like most of the default options. However, since we are working in a team, I feel like more controversial rules (such as `no-reduce`) should be discussed.

> But what is the sense to create named default export instead of just default export?

I would say readability, most likely.

---

I think that it's also worth mentioning, that XO doesn't only add additional rules with each release - when a controversial rule is given a strong NO from the community, it's removed. Maybe this will be the case with `unicorn/no-reduce`.

My current recommendation is:

If there are any "more readable" ways to replace `Array.reduce`, that do not require adding polyfills - we should use them. However, if the only good solution is `Object.fromEntries` - let's ignore this rule and maybe we could use it once our minimum Node.js version will be 12+ :laughing:

Reply:
I never use linters in my own project they just come up with stupid opinionated rules. I follow some basic style guide and do whatever I want! Everything I want linters to do is to spot error/misstakes deprecation (that sort of things) in my code not force me with stylistic rules. copy/pasting code between projects becomes a pain than.

[Linting, It's Pretty Much All Pain And No Gain](https://www.bennadel.com/blog/3312-after-3-months-of-javascript-linting-its-pretty-much-all-pain-and-no-gain.htm)

For instance I still see `var` as useful and empty `catch` block as useful... if you want to use reduce fine, go ahead!